src/cmd/oui.rs

```diff
@@ -362,15 +362,15 @@ fn print_update_txn(
         OutputFormat::Table => {
             ptable!(
                 ["Key", "Value"],
-                ["OUI", txn.oui],
+                ["oui", txn.oui],
```

Comment:
lol no... I meant in tables use normal capital cases and in **json**` make **all** table  keys lowercase